gradle.properties

```diff
@@ -1,2 +1,3 @@
 group = org.threadly
 version = 5.28-SNAPSHOT
+moduleName = org.threadly
```

Comment:
As best I can tell, sub-module names are not a problem, just conflicting names can be a problem.  So the plan is to use `org.threadly` here as that's the root package, then other threadly libraries will get things like `org.threadly.litesockets` or `org.threadly.litesockets.http` or something.  We will want to do this project by project, and where it makes sense maybe we can do package changes to keep things consistent.